app/Http/Controllers/Three/SignupsController.php

```diff
@@ -53,4 +53,21 @@ public function show(Request $request, $id)
 
         return $this->item($signup, 200, [], $this->transformer, $request->query('include'));
     }
+
+    /**
+     * Delete a signup.
+     *
+     * @param  \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $signupDeleted = Signup::findOrFail($id)->delete();
```

Comment:
I think we can assume this is always successful (it'll throw an exception otherwise, unless we were using custom model events). I'd also like to start using [route model binding](https://laravel.com/docs/5.5/routing#route-model-binding) for grabbing the relevant resource. Let's add the [respond](https://github.com/DoSomething/northstar/blob/755c26b51f72b36bf205585d447d7abbb6170a36/app/Http/Controllers/Traits/TransformsResponses.php#L123-L141) method from Northstar's `TransformRequests` in Rogue & switch this to:

```php
public function destroy(Signup $signup)
{
    $signup->delete();

    return $this->respond('Signup deleted.', 200);
}
```

Reply:
oh cool! That is a lot better to read!

Reply:
@DFurnes I _think_ I am doing everything correctly to use Northstar's `TransformRequest` but it looks like it can't find it:
```
{
    "error": {
        "code": 500,
        "message": "Trait 'Northstar\\Http\\Controllers\\Traits\\TransformsResponses' not found"
    },
    "debug": {
        "file": "/home/vagrant/DoSomething.org/rogue/app/Http/Controllers/Three/SignupsController.php",
        "line": 13
    }
}
```
I have to probably include that somehow into Rogue, right?

Reply:
We want to import Rogue's [`TransformRequests` trait](https://github.com/DoSomething/rogue/blob/master/app/Http/Controllers/Traits/TransformsRequests.php) here, you can't require classes across apps! We didn't copy `respond` when we made our version of that class here, so you'll need to copy-paste that from the Northstar source into the corresponding Rogue file.

Reply:
ah right! That's what I was thinking but then thought I was having a brain fart 🤦‍♂️ Also just realized I didn't read your comment correctly that explicitly states that - sorry!!